ui/imodel-components-react/src/imodel-components-react/timeline/SolarTimeline.scss

```diff
@@ -4,356 +4,136 @@
  *--------------------------------------------------------------------------------------------*/
 @import "~@itwin/core-react/lib/cjs/core-react/style/themecolors";
 
-$top-position: 6px;
-$bottom-position: 4px;
-
-.solar-slider.solar-timeline > .iui-slider-container {
-  height: 28px;
-}
-
+// #region Solar timeline
 .solar-timeline-wrapper {
-  box-sizing: border-box;
-  background: var(--iui-color-background);
   width: 100%;
-  display: flex;
-  flex-direction: column;
   position: relative;
-  padding: 8px 16px;
-  color: var(--iui-color-text);
-  font-size: var(--iui-font-size-1);
-
-  > .header {
-    display: flex;
-    align-items: center;
-    position: relative;
-
-    > .play-button {
-      width: 26px;
-      height: 26px;
-      background: inherit;
-      border: none;
-      font-size: 20px;
-    }
-
-    > .shadow-settings-button {
-      display: flex;
-      align-items: center;
-      color: var(--iui-color-icon);
-      width: 26px;
-      height: 26px;
-      background: inherit;
-      border: none;
-      font-size: 16px;
-      margin-left: 6px;
-      cursor: pointer;
-
-      &:hover {
-        color: var(--iui-color-icon-hover);
-      }
-
-      &:focus-visible {
-        outline: 1px solid var(--iui-color-border-accent);
-        outline-offset: -1px;
-      }
-    }
-
-    > .current-date {
-      min-width: 175px;
-      margin-left: 8px;
-      margin-right: 4px;
-      background: inherit;
-      padding: 3px 4px;
-      border: none;
-      display: flex;
-      align-items: center;
-      color: var(--iui-color-text);
-      font-size: var(--iui-font-size-1);
-      cursor: pointer;
-
-      &:focus-visible {
-        outline: 1px solid var(--iui-color-border-accent);
-        outline-offset: -1px;
-      }
-
-      > span:nth-child(2) {
-        margin: 0 6px;
-      }
-
-      > .icon-calendar {
-        flex-grow: 1;
-        margin-left: auto;
-      }
-    }
-
-    > .speed-container {
-      display: flex;
-      align-items: center;
-      margin-left: auto;
-      cursor: pointer;
-
-      > .speed {
-        margin-left: 12px;
-        width: 0;
-        opacity: 0;
-        transition: all 0.2s linear;
-        margin-right: 10px;
-      }
-    }
+  display: grid;
+  align-items: center;
+  grid-template-columns: auto 1fr auto;
+  grid-template-areas: "solar-timeline-start solar-timeline solar-timeline-end";
 
-    > .loop-playback {
-      color: var(--iui-color-icon-accent);
-      font-size: 16px;
-      cursor: pointer;
-
-      &:focus {
-        outline: none;
-      }
-    }
-
-    > .no-loop-playback {
-      color: var(--iui-color-icon);
-      font-size: 16px;
-      cursor: pointer;
-
-      &:hover {
-        color: var(--iui-color-icon-hover);
-      }
-
-      &:focus {
-        outline: none;
-      }
-    }
+  @media only screen and (max-width: 768px) {
```

Comment:
Should probably use a container query instead?

Reply:
Yes, that's a good idea.

Reply:
Using container queries now!  [If the user's browser doesn't support container queries](https://caniuse.com/css-container-queries), the fallback is the current implementation.